public/index.html

```diff
@@ -0,0 +1,12 @@
+<html>
+
+<head>
+    <title>Clickly blocky test</title>
+    <script type="text/javascript" src="dist/bundle.js"></script>
+</head>
+
+<body>
+    <div id="gridEl"></div>
+</body>
+
+</html>
```

Comment:
my vscode for linux auto-formatted this with the extra spaces too, it's quite upsetting, will fix.